.buildkite/release_pipeline.yaml

```diff
@@ -19,6 +19,14 @@ steps:
     command: .buildkite/scripts/build_component.sh hab
     agents:
       queue: habitat-release
+  
+  - label: ":linux: :see_no_evil: :habicat: core/hab/kernel2"
```

Comment:
I'd separate the "kernel2" text from the package name, just so it's very clear that "kernel2" isn't some kind of weird version string 😄

Reply:
Also, is kernel 2 "evil"? 🤣

Reply:
Good point,  I'll fix that.  

A little bit 😂  That just happened to be in front of me when I stopped looking for a good representative icon for the kernel2 builds.  It was a small delighter for myself at the end of the day 😄   It should change when we decide on an icon to use.

Reply:
I don't necessarily object 😄 I'm a big fan of using "delighter" emoji in Buildkite.